portality/view/forms.py

```diff
@@ -67,12 +67,15 @@ def __init__(self, *args, **kwargs):
         try:
             if not current_user.is_anonymous:
                 if "admin" in current_user.role and "id" in kwargs :
-                    issns = models.Journal.issns_by_owner(a.get_owner())
+                    eissns = models.Journal.eissns_by_owner(a.get_owner())
+                    pissns = models.Journal.pissns_by_owner(a.get_owner())
```

Comment:
Yeah, so, taking my approach above of querying for them both at the same time, we could introduce a function which returns a tuple, like:

```
eissns, pissns = models.Journal.issns_by_type_by_owner(a.get_owner())
```